archunit-junit/junit5/engine/build.gradle

```diff
@@ -12,7 +12,6 @@ dependencies {
     api project(path: ':archunit-junit5-engine-api')
     implementation project(path: ':archunit-junit')
     dependency.addGuava { dependencyNotation, config -> implementation(dependencyNotation, config) }
-    implementation dependency.slf4j
```

Comment:
I think this one here is not "unused" though, no? I would only remove it, if it's really not used, not just cause it "randomly" compiles by pulling in the dependency transitively? 🤔

Reply:
Oops, I don't know how I could miss the `Logger` in `ArchUnitTestDescriptor`... Thanks for checking thoroughly!